### ISSUE TITLE
[BUG] Roblox crashes within minutes of opening w/o errors

### ISSUE THREAD

**Issue by only1mistakE** [bloxstraplabs/bloxstrap#3120]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### What problem did you encounter?

Roblox crashes right after I open it. No errors or anything, literally nothing. Just freezes then hangs there for like around a minute and then exits on its own.

### Bloxstrap Log

[Bloxstrap_20241010T114102Z.log](https://github.com/user-attachments/files/17327449/Bloxstrap_20241010T114102Z.log)

**Comment by pizzaboxer** [comment]
Does the original launcher work?

**Comment by Zacgamingpro1234** [comment]
I'm guessing this is after installing windows 11? If yes follow [this](https://github.com/bloxstraplabs/bloxstrap/issues/3079#issuecomment-2395495261) and it will work first try.

**Comment by only1mistakE** [comment]
> Does the original launcher work?

Yeah, works fine. I don't have any FFlags aside from the already included ones that come with the bootstrapper.

**Comment by only1mistakE** [comment]
It has been found that this is an issue only for Windows 11 build 26100.2033, which somehow conflicts with Byfron and causes the game to crash. You may now close the thread.

**Comment by Zacgamingpro1234** [comment]
> It has been found that this is an issue only for Windows 11 build 26100.2033.

Windows just updated an app called Windows Application Compatibility Enhancements
![image](https://github.com/user-attachments/assets/14c7464b-064b-4782-8c07-d672a8c8af5e)
Does this app update fix the crash?

**Comment by only1mistakE** [comment]
> > It has been found that this is an issue only for Windows 11 build 26100.2033.
> 
> Windows just updated an app called Windows Application Compatibility Enhancements ![image](https://private-user-images.githubusercontent.com/140814307/375853538-14c7464b-064b-4782-8c07-d672a8c8af5e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2OTk0MjEsIm5iZiI6MTcyODY5OTEyMSwicGF0aCI6Ii8xNDA4MTQzMDcvMzc1ODUzNTM4LTE0Yzc0NjRiLTA2NGItNDc4Mi04YzA3LWQ2NzJhOGM4YWY1ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMlQwMjEyMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05OTdiYTgwZjZhZTc0OThlYjNiNmFmYTZhZDlhMjgwYjRhMTM2ODIzZjRjZWE3Mzc1ZWVkNDM1Nzg1OGE3OTZjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6NxtttRoJlb_Vs3mH8e8TtAq5ny1eENH5OmmriiMVP8) Does this app update fix the crash? (I'm on 23H2 so I can't test yet)

Can't even find it.

**Comment by Zacgamingpro1234** [comment]
Just updated to 24H2 and I can verify it's fixed, you should close this issue.

**Comment by only1mistakE** [comment]
> Just updated to 24H2 and I can verify it's fixed, you should close this issue.

Are you able to test if the issue can be replicated with MSI Afterburner? Just default settings, no OC.

**Comment by Zacgamingpro1234** [comment]
> Are you able to test if the issue can be replicated with MSI Afterburner? Just default settings, no OC.

Yes the issue can be replicated with MSI Afterburner, default settings. I don't see how hyperion/byfron would block these. Very weird.

**Comment by only1mistakE** [comment]
> > > Are you able to test if the issue can be replicated with MSI Afterburner? Just default settings, no OC.
> > 
> > Yes the issue can be replicated with MSI Afterburner, default settings. I don't see how hyperion/byfron would block these. Very weird.
> 
> rivatuner's capturing method flags hyperion, you either have to disable it or put the capturing method to the lowest from the settings

Thing is, this happens even without my RivaTuner installed.

**Comment by only1mistakE** [comment]
Update since it's quiet here:

It stopped happening for a while after I turned off MSI Afterburner start on boot. Fast forward to today and it's happening even w/o MSI Afterburner. This may or may not just be Hyperion flagging random stuff that it doesn't like as a false positive. If anyone's got the same problem just state it here, we'll find a way.

**Comment by Zacgamingpro1234** [comment]
Check if you can launch bloxstrap when booting into windows with safe boot.

**Comment by only1mistakE** [comment]
> Check if you can launch bloxstrap when booting into windows with safe boot.

I'll try it tomorrow. In the meantime if you have any other suggestions just spit em out, I'll try them all at once.

**Comment by pizzaboxer** [comment]
Just clarifying here, the original bootstrapper works fine, but you're only encountering issues with Bloxstrap?

**Comment by only1mistakE** [comment]
> Just clarifying here, the original bootstrapper works fine, but you're only encountering issues with Bloxstrap?

Yep.

**Comment by pizzaboxer** [comment]
Strange, I know you said you're on mostly default settings, but can I see your config? https://github.com/pizzaboxer/bloxstrap/wiki/Uploading-your-Bloxstrap-configuration

**Comment by only1mistakE** [comment]
> Strange, I know you said you're on mostly default settings, but can I see your config? https://github.com/pizzaboxer/bloxstrap/wiki/Uploading-your-Bloxstrap-configuration

[ClientAppSettings.json](https://github.com/user-attachments/files/17378008/ClientAppSettings.json)
[Settings.json](https://github.com/user-attachments/files/17378010/Settings.json)
[State.json](https://github.com/user-attachments/files/17378012/State.json)

Just been playing Redmatch 2 since yesterday. Not a plug but it's a cool game, anyways, until I can get to the root of this issue, I probably won't try launching Roblox that much.

**Comment by only1mistakE** [comment]
UPD: Roblox has replied to me with these instructions after I stated my issues.

"Please ensure that all Roblox files in the AppData directory are removed. To do this:

Uninstall Roblox from your computer
Open the Command Prompt. You can open it by pressing Win+R > Type ""CMD"" > Press Enter.

Type in the following commands:

cd AppData\Local (then press enter)
rmdir /s Roblox (then press enter)
Enter ""y"" when asked to answer (then press enter)
Restart your computer
Reinstall Roblox"

I'll give an update in around 30m - 1hr.

**Comment by 372seca** [comment]
> I'm guessing this is after installing windows 11? If yes follow [this](https://github.com/bloxstraplabs/bloxstrap/issues/3079#issuecomment-2395495261) and it will work first try.

i have those options on off...
